### DISCUSSION TITLE
Q-Input with v-model first readonly and then editable with save action

### DISCUSSION THREAD

**Discussion by Kellrkind** [quasarframework/quasar#13565] [General - Components / Directives / etc]
Hello together :)

I'm working on an application where the data is loaded from a server and is displayed in several q-expansation-items (v-for).
In this q-expansion-item I'm displaying the different object values as q-input fields in readonly mode. Looks like this:
 ```
<q-expansion-item
        v-for="dao in daoList"
        :key="dao"
        :label="dao.ort"
        :caption="formatDate(dao.datum)"
        expand-separator
        expand-icon-class="positive"
        :value="dao.opened"
        :header-class="{
          'header-blue': dao.opened
        }"
        :content-inset-level="1"
        popup>

      <q-separator/>

      <div class="q-gutter-md row items-start">
        <q-input style="width: 15%" :disable="!dao.editMode" :readonly="!dao.editMode" v-model="dao.ort" label="Ort *" stack-label
                 ref="newDaoOrtRef"
                 lazy-rules/>
      </div>

      <div class="q-gutter-md row">
        <q-input style="width: 15%" :disable="!dao.editMode" :readonly="!dao.editMode" v-model="dao.abfahrt" type="time" label="Abfahrt"
                 stack-label/>
        <q-input style="width: 15%" :disable="!dao.editMode" :readonly="!dao.editMode" v-model="beginn" type="time"
                 label="Beginn" stack-label/>
        <q-input style="width: 15%" :disable="!dao.editMode" :readonly="!dao.editMode" v-model="dao.treffpunkt" type="time"
                 label="Treffpunkt"
                 stack-label/>
        <q-checkbox disable readonly left-label v-model="dao.ue18" label="ü18" checked-icon="task_alt" unchecked-icon="highlight_off"/>
      </div>

      <q-btn class="daoButtonStyle" round color="blue" icon="edit" @click="onEdit(dao)"/>
      <q-btn class="daoButtonStyle" v-show="!!dao.editMode" round color="blue" icon="save" @click="onSave(dao)"/>
      <q-btn class="daoButtonStyle" round color="blue" icon="delete" @click="onDelete(dao)"/>

    </q-expansion-item>
```

Now I want to give the user the opportunity to edit the stored data (see the 3 three buttons at the end of the code)
I've tried it this way:
- Underneath the displayed columns is an edit button. When the user clicks it, all readonly properties are disabled, the edit button disappears and a save button appears. Works totally fine.
- It should now be possible to edit the different fields and store it at the database via an axios call. The updated object should now be added to my vuex store and the user should be happy with his updated object.

The problem now is, that every action in an disabled readonly field is directly reverted to the initial value. I can't even hit the save button fast enough 😅 

Do you have any recommendations if my way of solving this case possible? And when yes, how do I solve my problem?
Or do I have to use another technical solution?

Thank you :)

**Comment by metalsadman** [comment]
make a reproduction of that here https://stackblitz.com/fork/quasarframework
